libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -914,24 +909,21 @@ void AC_PosControl::write_log()
 // @Field: TAY: Target acceleration, Y-axis
 // @Field: AX: Acceleration, X-axis
 // @Field: AY: Acceleration, Y-axis
-    AP::logger().Write("PSC",
-                       "TimeUS,TPX,TPY,PX,PY,TVX,TVY,VX,VY,TAX,TAY,AX,AY",
-                       "smmmmnnnnoooo",
-                       "F000000000000",
-                       "Qffffffffffff",
-                       AP_HAL::micros64(),
-                       double(pos_target.x * 0.01f),
-                       double(pos_target.y * 0.01f),
-                       double(position.x * 0.01f),
-                       double(position.y * 0.01f),
-                       double(vel_target.x * 0.01f),
-                       double(vel_target.y * 0.01f),
-                       double(velocity.x * 0.01f),
-                       double(velocity.y * 0.01f),
-                       double(accel_target.x * 0.01f),
-                       double(accel_target.y * 0.01f),
-                       double(accel_x * 0.01f),
-                       double(accel_y * 0.01f));
+   AP::logger().Write_PSC(get_pos_target(), _inav.get_position(), get_vel_target(), _inav.get_velocity(), get_accel_target(), accel_x, accel_y);
+
+// @LoggerMessage: PSCP
```

Comment:
This documentation has traditionally gone in LogStructure.h on the basis that the call to `Write_PSCP` could be made elsewhere.

Reply:
done